ambari-server/src/main/java/org/apache/ambari/server/controller/AmbariHandlerList.java

```diff
@@ -251,6 +251,10 @@ private WebAppContext getHandler(ViewInstanceEntity viewInstanceDefinition)
     webAppContext.addFilter(new FilterHolder(springSecurityFilter), "/*", AmbariServer.DISPATCHER_TYPES);
     webAppContext.setAllowNullPathInfo(true);
 
+    if (webAppContext.getErrorHandler() != null) {
+      webAppContext.getErrorHandler().setShowStacks(false);
```

Comment:
can we make this configurable via introducing some debug property  or add logging to ambari-server.log? 

In most cases, such kind of errors r not present in the logs and the only source of them r response from the server

Reply:
yeah sure, it makes sense to make it configurable. Let me check if we already have this type of property or if I need to create a new one.

Reply:
I couldn't find a config property that I could use so I introduced a new one to display the error stacks. @hapylestat  please review it